.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,756 @@
+3. **Implementation:** 
+    - Make sure you have read: "[Before you get coding](#before-you-get-coding)".
+    - Signal to others you are working on the issue by assigning yourself.
+    - Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](#branch). 
+    - Create a test that reproduces the bug following guidelines in: "[The good: Test](#test)". 
+    - Write code to fix the bug
+4. **Follow steps 4 and 5 in "[Checklist: I have a component request](#checklist-i-have-a-component-request)".**
+
+# Setup environment and tooling for contribution
+
+<!-- The following section is responsible for teaching people how to get setup with 
+the repo such that they can get coding -->
+This section will help you get going with Kirby such that you can begin contributing. 
+
+## Clone and install the repository
+
+<!-- how to clone and get going with the kirby repo-->
+**Note:** make sure you have the right version of Node installed before installing the repository (see: [Node](#node)).
+
+1. Clone the repository: 
+
+``` 
+
+git clone https://github.com/kirbydesign/designsystem
+```
+
+2. Install dependencies: 
+
+``` 
+
+npm install
+```
+
+3. Start the cookbook server:
+
+``` 
+
+npm start
+```
+
+You should now be able to open `http://localhost:4200` in a browser to access the cookbook.
+
+## Node 
+
+<!-- Which version should people use, and perhaps a recommendation on which tool people can use to manage versions. -->
+[Node](https://nodejs.org/en/) is the runtime used for development. 
+
+It is recommended to use `v12` for development as we currently do not support node LTS (see issue: [#1443](https://github.com/kirbydesign/designsystem/issues/1443)). 
+
+If you need to have multiple versions of node installed then a version manager such as [NVM](https://github.com/nvm-sh/nvm) can be used. 
+
+## TSlint 
+
+<!-- People should make sure to have it enabled to avoid unnecessary comments during code review-->
+**Note:** Work is being done to migrate from TSLint to ESLint (see issue: [#1317](https://github.com/kirbydesign/designsystem/issues/1317)). 
+
+TSLint is used for linting which helps keep up the quality of the submitted code for Kirby. 
+
+Scripts for running TSLint can be found in `package.json` . We however recommend that you install a TSLint plugin for your editor. It can be useful to enable "fix on save"-functionality such that simple errors are fixed automatically. 
+
+**Useful resources:**
+
+* [TSLint extension for VSCode](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin).
+* [Enabling "fix on save" in VSCode](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin#auto-fix-and-auto-fix-on-save).
+
+### VSCode "workspace library execution"-issue
```

Comment:
Is this still an issue?

Reply:
Yes, I just got it this morning.

Reply:
Should we add `.vscode/settings.json` to `.gitignore`?